include/nbl/ext/RadixSort/RadixSort.h

```diff
@@ -0,0 +1,112 @@
+#ifndef _NBL_EXT_RADIXSORT_INCLUDED_
+
+#include "nabla.h"
+
+namespace nbl
+{
+namespace ext
+{
+namespace RadixSort
+{
+
+typedef uint32_t uint;
+#include "nbl/builtin/glsl/ext/RadixSort/parameters_struct.glsl"
+
+class RadixSort final : public core::IReferenceCounted
+{
+public:
+	_NBL_STATIC_INLINE_CONSTEXPR uint32_t DEFAULT_WORKGROUP_SIZE = 256u;
+	_NBL_STATIC_INLINE_CONSTEXPR uint32_t BITS_PER_PASS = 4u;
+	_NBL_STATIC_INLINE_CONSTEXPR uint32_t PASS_COUNT = 32u / BITS_PER_PASS;
+	_NBL_STATIC_INLINE_CONSTEXPR uint32_t BUCKETS_COUNT = 1 << BITS_PER_PASS;
+
+	typedef nbl_glsl_ext_RadixSort_Parameters_t Parameters_t;
+
+	struct DispatchInfo_t
+	{
+		uint32_t wg_count;
```

Comment:
Regarding `wg_count` it should be a `[3]` reasoning is same as https://github.com/Devsh-Graphics-Programming/Nabla/pull/75/files#r608113556